paddle/fluid/framework/executor.cc

```diff
@@ -557,7 +563,6 @@ void Executor::EnableMKLDNN(const ProgramDesc& program) {
       }
     }
   }
-  platform::AttachPointerHashToMKLDNNKey(this, place_);
```

Comment:
Why you have moved out this call?

Reply:
@arogowie-intel  Because it is now called independently from EnableMKLDNN. EnableMKLDNN is just setting "use_mkldnn=True" in ops. But you can set manualy this attribute without EnableMKLDNN , so there were sitatuation when oneDNN is used but not EnableMKLDNN is called.